docs/Usage.md

```diff
@@ -83,6 +82,14 @@ To activate the data transfer from the Modbus TCP Connector, proceed as followin
 
 - go to tab 'Assets & Connectivity' and add the variables, that were configured within the Modbus TCP Connector
 
-![DataServiceAdapter](/docs/graphics/DataService_Add.png)
+- ## Use Store Data in Common Configurator

```

Comment:
From this part, it should be clarified that, to use "Store Data" option, IIH Essentials needs to be integrated into Common Configurator. Therefore, previous step "Use IIH Essentials" is not necessary.